### PR TITLE
Feature fast flux

### PR DESCRIPTION
**Pull request by grburgess**
Some doc fix ups and a faster way to compute fluxes.

### CODE REVIEW

File: docs/md_docs/flux_examples.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+<!-- #region -->
+# Point Source Fluxes and Multiple Sources
+
+Once one has computed a spectral to a point source, getting the flux of that source is possible.
```
Comment (henrikef) [inline docs/md_docs/flux_examples.md:19]: Nitpicky, but we can get (spatially integrated) fluxes from extended sources as well.

File: docs/md_docs/flux_examples.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+Lets grab the result. Rememer, we can save the results to disk, so all of the following ooperations can be run at a later time without having to redo all the above steps!
+
+```python
+result = ba.results
+result.corner_plot();
+```
+
+## Computing fluxes
+
+Now we will compute fluxes. We can compute them an many different units, over any energy range also specified in any units.
```
Comment (henrikef) [inline docs/md_docs/flux_examples.md:213]: Suggestion: Mention explicitly that we can calculate photon fluxes (in units of photons / time/ area), energy fluxes (energy/time/area) or nuFnu fluxes ( energy^2 / time /area [????] ), and the user can pick arbitrary units for energy/area/time in each case.

**Review by henrikef — APPROVED** [review]
I think this is good to go, I do have some suggestions for the docs but these aren't critical.

File: docs/md_docs/flux_examples.md (hunk 1 above)
Comment (grburgess) [reply docs/md_docs/flux_examples.md:19]: should we make a link to some docs on that?

Comment (henrikef) [reply docs/md_docs/flux_examples.md:19]: Or, just change point source to "point source and extended source" for now :D

**Pull request merged**